spec/models/miq_action_spec.rb

```diff
@@ -159,4 +159,44 @@
       end
     end
   end
+
+  context '.create_default_actions' do
+    context 'miq_actions table is initially empty' do
+      before { MiqAction.destroy_all }
```

Comment:
This context is not needed since the tests run in transactions and the db gets rolled back so the table is empty to begin with.